hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/raft/RaftSharedContext.java

```diff
@@ -270,7 +272,15 @@ protected void notifyCache(String action, HugeType type, Object id) {
         }
         try {
             // How to avoid update cache from server info
-            eventHub.notify(Events.CACHE, action, type, id);
+            if (ids == null) {
+                eventHub.call(Events.CACHE, action, type);
+            } else {
+                if (ids.size() == 1) {
+                    eventHub.call(Events.CACHE, action, type, ids.get(0));
+                } else {
+                    eventHub.call(Events.CACHE, action, type, ids.toArray());
```

Comment:
seems can delete if-branch

Reply:
Can't delete.

In CachedSchemaTransaction 

```
if (Cache.ACTION_INVALID.equals(args[0])) {
                event.checkArgs(String.class, HugeType.class, Id.class);
```

the 3rd param must be Id class.

In CachedGraphTransaction

```
if (Cache.ACTION_INVALID.equals(args[0])) {
                event.checkArgs(String.class, HugeType.class, Object.class);
```

the 3rd param can be Id or Id[] class